azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,50 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge devices"] = """
+        type: group
+        short-summary: Commands to create and manage IoT Edge devices.
+    """
+
+    helps["iot edge devices create"] = """
+        type: command
+        short-summary: Create configuratations of edge devices in an IoT Hub.
```

Comment:
creating configurations of edge devices != creating edge devices, right?

Reply:
Yeah I can see how this could be confused with "Edge Configurations" but this is creating... (not hierarchies, but... topologies?)

I need to think of a better word than "collection", "group", or "a bunch of"

Reply:
Why is "Create edge devices in an IoT Hub" not good? Otherwise collection would be ok or even multiple devices. Group can be confused with device groups

Reply:
Why is "Create edge devices in an IoT Hub" not good? Otherwise collection would be ok or even multiple devices (instead of a bunch of). Group can be confused with device groups...